code/students-performance/src/main/java/ru/unn/agile/StudentsPerformance/Model/PerformanceCalculator.java

```diff
@@ -0,0 +1,63 @@
+package ru.unn.agile.StudentsPerformance.Model;
```

Comment:
Could you rename all occurrences "Number" to "Count"? I guess it's more readable. Up to you.

Reply:
Done